opencog/atoms/execution/ExecutionOutputLink.cc

```diff
@@ -261,7 +266,7 @@ void LibraryManager::setLocalFunc(std::string libName, std::string funcName, voi
     _functions[funcID] = func;
 }
 
-void opencog::setLocalFunc(std::string funcName, Handle* (*func)(AtomSpace *, Handle*))
+void opencog::setLocalSchema(std::string funcName, Handle* (*func)(AtomSpace *, Handle*))
 {
     LibraryManager::setLocalFunc("", funcName, reinterpret_cast<void*>(func));
```

Comment:
It looks like the indentation type of this file is tab, while you're using space.